views/bootstrapindex.ejs

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!---This is The color mode of the page-->
+    <script src="public/js/color-modes.js"></script>
+    <!--End of color mode-->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap demo</title>
+    <link rel="stylesheet" href="public/css/dashboard.css">
```

Comment:
the main bootstrap css should be added first before we add custom css. **dashboard.css** is a custom css that overrides the styling rules defined inside bootstrap.min.css, that's why.